geoips/plugins/modules/readers/amsr2_netcdf.py

```diff
@@ -285,7 +285,14 @@ def read_amsr_data(full_xarray, chans):
     return xarrays
 
 
-def call(fnames, metadata_only=False, chans=None, area_def=None, self_register=False):
+def call(
+    fnames,
+    metadata_only=False,
+    chans=None,
+    area_def=None,
+    self_register=False,
+    test_arg="AMSR2 Default Test Arg",
```

Comment:
Updated both AMSR2 and AHI readers, to test a foreground and background read.

Reply:
Can we move this test to a unit test? It would be good to have unit tests for the readers anyway and we could test the various different arguments.